frontend/src/lib/utils.ts

```diff
@@ -799,6 +799,26 @@ export type Value = {
 	[key: string]: any
 }
 
+export	function replaceFalseWithUndefined(obj: any) {
```

Comment:
you don't want to modify in place the original object, split it into a recursive function and do a structuredClone prior to calling the rec function